plugin.video.iplayerwww/addon.xml

```diff
@@ -12,7 +12,7 @@
     <description lang="en">This add-on enables playing of Live and Catchup TV (UK only) and Radio content (UK or international) from the BBC iPlayer website. It is not created, maintained or in any way affiliated with the BBC. The add-on only provides an interface to access the BBC iPlayer website from Kodi.</description>
     <platform>all</platform>
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
-    <forum>http://forum.kodi.tv/showthread.php?tid=239378</forum>
+    <forum>https://forum.kodi.tv/showthread.php?tid=353349</forum>
```

Comment:
At some point please adjust the license format to a spdx compliant identifier (GPL-2.0-only or GPL-2.0-or-later). Not needed for now, but we might request it when later this is submitted to matrix